introduction.md

```diff
@@ -9,3 +9,11 @@ No sense word in freeeedom
 etc etc etc
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?